components/dm/spec/logic.go

```diff
@@ -216,6 +221,9 @@ func (c *DMWorkerComponent) Instances() []Instance {
 					s.DataDir,
 				},
 				StatusFn: s.Status,
+				UptimeFn: func(tlsCfg *tls.Config) time.Duration {
```

Comment:
This is actually "Last State Time"

Reply:
IMO, the Last State Time is the last timestamp we saw the state , but what I want to point is the alive time duration of the process.